superset/assets/webpack.config.js

```diff
@@ -108,6 +108,14 @@ const config = {
         include: APP_DIR + '/node_modules/mapbox-gl/js/render/painter/use_program.js',
         loader: 'transform/cacheable?brfs',
       },
+      {
+        test: /\.po$/,
+        loader: 'po-catalog-loader',
```

Comment:
This is the PO file output that is parsed through Jed method. Jed requires po-catalog-loader, and I'm not sure if Babel can